server/plugin/plugin.go

```diff
@@ -97,9 +97,9 @@ func (p *plugin) ExecuteCommand(_ *mattermostPlugin.Context, commandArgs *model.
 	commandHandler := command.NewHandler(commandArgs, p.GetConfiguration(), p.sp)
 	args := strings.Fields(commandArgs.Command)
 
-	commandResponse, err := commandHandler.Handle(args...)
+	commandResponse, err := commandHandler.Handle(commandArgs.UserId, args...)
 	if err == nil {
-		return p.sendEphemeralResponse(commandArgs, commandResponse.Text), nil
+		return commandResponse, nil
```

Comment:
It seems like no `model.CommandResponse` is used in the plugin any longer. Why does `postCommandResponse` still return a ` model.CommandResponse`?

Reply:
@hanzei I think it's because `ExecuteCommand` expects that for its return value

Reply:
Well, if the value returned in `ExecuteCommand` is always `&model.CommandResponse{}` we can hard-code this.

Reply:
`Handle` could also return a `string` which is then used as an ephemeral message similar to how it's done in the GitHub plugin: https://github.com/mattermost/mattermost-plugin-github/blob/a60ac1934e08e89e49535b9822f47352584569db/server/plugin/command.go#L599-L603

Reply:
@hanzei @mickmister have made the required changes can you Please check.